app/main.py

```diff
@@ -1,2 +1,98 @@
+from __future__ import annotations
+from typing import Hashable, Any, TypeAlias, Iterator
+from dataclasses import dataclass
+
+Key: TypeAlias = Hashable
+Value: TypeAlias = Any
+
+INITIAL_CAPACITY = 8
+RESIZE_THRESHOLD = 2 / 3
```

Comment:
The `RESIZE_THRESHOLD` is defined as `2 / 3`, which results in a float value. If the intention is to use a fraction, consider using `fractions.Fraction` or define it as a float directly, e.g., `0.66`.